attempt_6_animals/simple-reference-animals.squiggle

```diff
@@ -126,16 +111,260 @@ improvement_as_proportion_of_lifetime = 0.05 to 0.5
+mfa_item = {
+  name: "Mercy For Animals (2022)",
+  value: value_mfa -> ss
+}
+
+// New Harvest funds research in cellular agriculture, i.e., the development of animal products using cells instead of animals. Through grant programs, they fund graduate and postdoctoral research projects in the field of cellular agriculture, as well as undergraduate and master-level projects for researchers-in-training. They also carry out public engagement and community-building activities via their podcast, blog, research publications, and research summaries. New Harvest was selected as an ACE Standout Charity in both December 2015 and November 2021. Read Review.
+value_new_harvest = gfi_item.value * tt(0.01 to 0.5)
+new_harvest_item = {
+  name: "New Harvest (2022)",
+  value: value_new_harvest -> ss
+}
+
+// Sinergia Animal operates in Indonesia, Thailand, Argentina, Brazil, Colombia, Uruguay, Chile, Ecuador, and Peru. They work to improve farmed animal welfare standards, increase the availability of animal-free products, decrease the consumption of animal products, and strengthen the animal advocacy movement. Sinergia Animal engages in corporate outreach to secure animal welfare commitments from major retailers. They also engage in investor and media outreach, policy work, investigations, individual and producer outreach, institutional outreach, and research. Sinergia Animal has been an ACE Standout Charity since November 2018. Read Review.
+value_sinergia_animal = value_reference_top_animal_org * tt(0.05 to 20)
+sinergia_animal_item = {
+  name: "Sinergia Animal (2022)",
+  value: value_sinergia_animal -> ss
+}
+
+//
+//xiaobuVEGAN is a China-based organization working to reduce farmed animal suffering. Their work aims to increase the availability of animal-free products, strengthen the animal advocacy movement, and decrease the consumption of animal products. xiaobuVEGAN engages in institutional outreach, runs an app that provides resources about vegan options in China, and supports dietary change via their vegan challenge program. xiaobuVEGAN has been an ACE Standout Charity since November 2021. Read Review.
+value_xiaobuVEGAN = value_reference_top_animal_org * truncateRight(mixture([0.1 to 10, 1 to 100], [7/10, 3/10]), 1000)
+xiaobuVEGAN_item = {
+  name: "xiaobuVEGAN (2022)",
+  value: value_xiaobuVEGAN -> ss
+}
+
+
+// Construct relative values object
+
+// animal estimates
+toAllAnimalEstimates = {|animal|
+  value_year_in_human_qalys = animal.value_year_in_human_qalys
+  value_in_lifetime_human_qalys = animal.lifetime * animal.value_year_in_human_qalys
+  value_per_calorie_human_qalys = animal.lifetime * animal.value_year_in_human_qalys / 
+    animal.calories
+
+  result = [
+    ({name: animal.name + " value per year of life", value: value_year_in_human_qalys}),
+    ({name: animal.name + " value per lifetime", value: value_in_lifetime_human_qalys}),
+    ({name: animal.name + " value per calorie", value: value_per_calorie_human_qalys}),    
```

Comment:
IDs follow certain naming conventions, they don't have spaces and stuff. You can see all the IDs [here](https://relative-values-git-develop-quantified-uncertainty.vercel.app/interfaces/macrovariables/about).

Reply:
Don't worry about having nice names, in this section. You just need to match IDs. Names are separately attached to IDs.

Reply:
Ok, adding some really very lazy ids.

Reply:
Added some better ids now.